pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -69,9 +104,28 @@ class PokemonDTO {
     this.baseStats = this.calculateBaseStats();
   }
 
+  /**
+   * Returns Pokemon data in consumable form (strings) for a calling view.
+   */
+  public getDisplayStats(): pokemonDisplayObj {
+    return {
+      name: this.name,
+      id: this.dexId.toString(),
+      moves: this.moves.map((moveEntry) => moveEntry.name),
+      type1: this.type1,
+      type2: this.type2,
+      sprites: {
+        frontDefault: this.frontDefault,
+        frontShiny: this.frontShiny
+      }
+    }
+  }
+
+  /**
+   * Calculates the 'base stat total' (BST) for this Pokemon.
+   */
   private calculateBaseStats(): number {
-    if (this.stats)
-    {
+    if (this.stats) {
       const s = this.stats;
       return s.hp + s.attack + s.defense + s.spAttack + s.spDefense + s.speed;
     }
```

Comment:
Oh right, we should probably formalize a styleguide, I think I'm guilty of switching between various styles

Reply:
Same, but I usually rely on whatever format Prettier is doing.